core/integrations/hubot/robot.js

```diff
@@ -0,0 +1,52 @@
+﻿var instance,
+    messageCallback,
+    shim = require.once('../../shim.js');
+
+exports.logger = {
+    error: console.error,
+    warning: console.warn,
+    info: console.info,
+    debug: console.debug
+};
+
+exports.receive = function (message) {
+    var msg = shim.createEvent(message.room, message.user.name, message.user.id, message.text);
+
+    var api = shim.createPlatformModule({
+        commandPrefix: exports.config.commandPrefix ? exports.config.commandPrefix : '/',
+        sendMessage: function(message, thread) {
+            var m = {
+                room: thread
+            };
+            instance.send(m, message);
+        }
+    });
+
+    messageCallback(api, msg);
+};
+
+exports.start = function (callback) {
+    messageCallback = callback;
+    instance.run();
+};
+
+exports.stop = function () {
+    instance.close();
+};
+
+exports.use = function (inst) {
+    instance = inst.use(exports);
+    return exports;
+};
+
+exports.brain = {
+    data: exports.config,
+    userForId: function() {
+        return {
+            id: 'foo',
+            name: 'bar'
+        }
+    }
+};
+
+exports.name = "Kassy";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530929734/issues/source?bid=3282987&fileBranchId=3283570#l52)